src/frontend/js/actions/api.js

```diff
@@ -0,0 +1,21 @@
+import { CALL_API } from 'redux-api-middleware';
+
+const API_URL = 'http://localhost:1337/api/';
+
+export const USERS_ENDPOINT = 'users';
+
+export const APIcall = (options) => {
```

Comment:
All the setup below is unnecessary with `redux-promise-middleware`, you just use `fetch` api with a generic function for handling http codes if necessary, but this looks nice if you need more control over how you're calling the APIs.

My concern is that for most cases this seems verbose